assets/src/amp-validation/amp-validated-url-post-edit-screen.js

```diff
@@ -46,10 +46,12 @@ const addBeforeUnloadPrompt = () => {
 	if ( beforeUnloadPromptAdded ) {
 		return;
 	}
+	// eslint-disable-next-line @wordpress/no-global-event-listener
 	window.addEventListener( 'beforeunload', onBeforeUnload );
```

Comment:
You should be able to use `global` here since the file is being transpiled via Webpack, which would then provide reference to `window`.

Reply:
Good points. Might as well use `global` in all these instances. Updated in 747061f52